_docs/quick-start-bmi.md

```diff
@@ -23,7 +23,7 @@ Before we start, we need to add some common dependencies in the parent project o
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>io.servicecomb</groupId>
+        <groupId>org.apache.servicecomb</groupId>
         <artifactId>java-chassis-dependencies</artifactId>
         <version>0.5.0</version>
```

Comment:
0.5.0 the package name of servicecomb is still `io.servicecomb`. We change it since 0.6.0-SNAPSHOT.